applications/stepswitch/src/stepswitch_resources.erl

```diff
@@ -1084,6 +1093,11 @@ gateway_from_jobj(JObj, #resrc{is_emergency=IsEmergency
             ,privacy_mode=kz_json:get_value(<<"privacy_mode">>, JObj, PrivacyMode)
             }.
 
+-spec gateway_rtcp_mux(kz_json:object()) -> boolean().
+gateway_rtcp_mux(JObj) ->
+    Default = kapps_config:get_is_true(?SS_CONFIG_CAT, <<"default_rtcp_mux">>, 'true'),
```

Comment:
@lazedo is required to add default value to `applications/stepswitch/src/stepswitch.hrl` file